### ISSUE TITLE
`information_schema.processlist` result is inconsistent with `show processlist`

### ISSUE THREAD

**Issue by ricardoreiter** [dolthub/dolt#6023]
https://github.com/jeremydaly/serverless-mysql Uses `SELECT * FROM information_schema.processlist` to retrieve the list of open connections. Dolt seems to return slightly different results from `information_schema.processlist` vs `show processlist`:

`show processlist`:
```
ID	USER	HOST	DB	COMMAND	TIME	STATE	INFO
1884	root	%	NULL	Query	0	running	show processlist
4560	root	%	NULL	Sleep	18		
```

`SELECT * FROM information_schema.processlist`:
```
ID	USER	HOST	DB			COMMAND	TIME	STATE	INFO
1884	root	%	authoring_consolidated	Query	0	processlist(processlist (0/? partitions))	SELECT * FROM
4560	root	%	authoring_consolidated	Query	54	running	
```

In the example above, ID 4560 should be sleeping as correctly returned by `show processlist`, but it stays as `Query` in the `information_schema.processlist` table.

**Comment by timsehn** [comment]
@jennifersp will fix this when she finishes up events.

**Comment by JCOR11599** [comment]
FIx for this has been merged in, and is in dolt release `v1.6.0`
